### PR TITLE
Create user interface for My notes application

### PR DESCRIPTION
**Pull request by danimarce**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            <ul>
+                <li><a href="https://developer.mozilla.org/en-US/">MDN Web Docs</a></li>
+                <li><a href="https://css-tricks.com/">css-tricks</a></li>
+                <li><a href="https://github.com/danimarce">My Github</a></li>
+            </ul>
+        </aside>
+    </main>
+    <footer>&copy2024 My Notes Application</footer>
+</body>
+</html>
```
Comment (tictools) [inline index.html]: Format code: empty line at the end of the file.

File: sanitize.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```
Comment (tictools) [inline sanitize.css]: Format code: empty line at the end of the file.

Why are u resetting box-sizing? 🤔

File: style.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    justify-content: center;
+}
+
+main {
+    margin: auto;
+    color: #333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 84.4vh;
```
Comment (tictools) [inline style.css]: Using vh can be dangerous. Do you know why?

Better use px or rem for the moment.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    gap: 1%;
```
Comment (tictools) [inline style.css]: Using % can be dangerous. Do you know why?

Better use px or rem for the moment.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: #ff5722;
+    color: #fff;
+    border: 1px solid #0001;
+    border-radius: 3px;
+    font-family: calibri;
+}
+
+input[type="submit"]:hover {
+    background-color: #e64a19;
+}
```
Comment (tictools) [inline style.css]: Format code: empty line at the end of the file.

**Review by tictools — CHANGES_REQUESTED** [review]

File: sanitize.css (hunk 2 above)
Comment (danimarce) [reply sanitize.css]: For the containers and elements to occupy exactly what I specify in the CSS.

File: style.css (hunk 4 above)
Comment (danimarce) [reply style.css]: I think depending on the size of the parent container the spacing between elements could be different.

**Pull request merged**